src/locationTracker/Location.tsx

```diff
@@ -70,6 +73,28 @@ function CheckLocation({ id }: { id: string }) {
         [id, show],
     );
 
+    const handleDragOver = (event: React.DragEvent<HTMLDivElement>) => {
+        event.preventDefault();
+        event.dataTransfer.dropEffect = 'move';
+    };
+
+    const handleDrop = (event: React.DragEvent<HTMLDivElement>) => {
+        event.preventDefault();
+        const itemName = event.dataTransfer.getData('text/plain');
+        handleItemDrag({ item: itemName });
+    };
```

Comment:
You marked some things as `draggable={false}`, but the user can still drag other things - highlighted text, files from their desktop, boss/dungeon images. An extracted helper function that checks that the data refers to a known inventory item would be good.